### PR TITLE
Force user login

### PR DESCRIPTION
**Pull request by brendandeere**
We highly recoomend dissallowing guest checkout for orders containing
subscription line items

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -72,7 +72,23 @@ To process actionable subscriptions simply run:
 
 To schedule this task we suggest using the [Whenever](https://github.com/javan/whenever) gem.
 
-# Testing
+### Guest Checkout
+
+Although it is possible to allow purchasing subscriptions through the guest
+checkout, there is no api to allow a user to manage those subscriptions after
```
Comment (alexblackie) [inline README.md]: api -> API

File: README.md [hunk 2]
```diff
@@ -72,7 +72,23 @@ To process actionable subscriptions simply run:
...
+they have been purchased.
+
+Because of this we **highly** recommend disabling guest checkout for carts
+which contain subscription_line_items.
```
Comment (alexblackie) [inline README.md]: subscription_line_item -> `subscription_line_items`

**Comment by brendandeere** [comment]
This also need to change because it technically wont work without a user since we validate the user on the subscription

**Comment by brendandeere** [comment]
@alexblackie Fixed. Also changed this to requiring guest checkout be disabled. Since we do validate the presence of a user on the subscription model

**Pull request merged**